Task 1/CHEFEZQ/Question.md

```diff
@@ -0,0 +1,34 @@
+Chef published a blog post, and is now receiving many queries about it. On day i, he receives Qi queries. But Chef can answer at most k queries in a single day.
+
+Chef always answers the maximum number of questions that he can on any given day (note however that this cannot be more than k). The remaining questions (if any) will be carried over to the next day.
+
+Fortunately, after n days, the queries have stopped. Chef would like to know the first day during which he has some free time, i.e. the first day when he answered less than k questions.
+
+Input:
```

Comment:
You can use the markdown heading formatting for sub headings. Something like this:

```md
#### Input
```
which would render as:

#### Input